code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -518,14 +518,28 @@
 
 	QDEL_LIST(temp_objs)
 
-	// Chance for souvenir or bruises
+	// Injuries due to questioning and souvenirs
 	if(prob(RETURN_SOUVENIR_CHANCE))
 		to_chat(M, "<span class='notice'>Your captors left you a souvenir for your troubles!</span>")
 		var/obj/item/souvenir = pick(souvenirs)
 		new souvenir(closet)
-	else if(prob(RETURN_BRUISE_CHANCE) && M.health >= 50)
-		to_chat(M, "<span class='warning'>You were roughed up a little by your captors before being sent back!</span>")
-		M.adjustBruteLoss(RETURN_BRUISE_DAMAGE)
+	if(prob(RETURN_INJURY_CHANCE) && M.health >= 50)
+		var/obj/item/organ/external/injury_target
+		to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back!</span>")
+		if(prob(20)) //remove a limb
+			if(prob(50))
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND,BODY_ZONE_PRECISE_L_HAND,BODY_ZONE_PRECISE_R_FOOT,BODY_ZONE_PRECISE_L_FOOT))
+				injury_target.droplimb()
+		else //fracture
+			if(ismachineperson(M) || isslime(M)) //They dont have bones, will need something here
```

Comment:
I'd suggest checking traits, like `if(HAS_TRAIT(M, TRAIT_NO_BONES))` and `if(<organ>.limb_flags & CANNOT_BREAK)`. Might help things be a bit more consistent.

Reply:
Would it be more sane/less snowflakey to do trait checks rather than tailor responses to IPCs/Slimes (as I think theyre the only outliers for bone breaks)